File: Java/Q6.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the dividend: ");
        int dividend = sc.nextInt();
        System.out.print("Enter the divisor: ");
        int divisor = sc.nextInt();

        System.out.println("Quotient = " + (dividend / divisor));
        System.out.println("Remainder = " + (dividend % divisor));

        sc.close();
    }
}

File: C++/Q6.cpp
#include <iostream>
using namespace std;

int main() {
    int dividend, divisor;
    cout << "Enter dividend and divisor: ";
    cin >> dividend >> divisor;
    cout << "Quotient = " << dividend / divisor << endl;
    cout << "Remainder = " << dividend % divisor;
    return 0;
}
